Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -129,6 +129,32 @@ public void SetBounds(Vector2 newBounds)
             bounds = newBounds;
         }
 
+        /// <summary>
```

Comment:
Also @julenka, what do you think of the weird OnValidate in here?

Reply:
I think it's out of scope for this PR, but would be good to probably make a custom inspector.